pkg/uiserver/uiserver.go

```diff
@@ -38,3 +42,52 @@ func Handler() http.Handler {
 func AssetFS() *assetfs.AssetFS {
 	return fs
 }
+
+type gzipHandler struct {
```

Comment:
Thanks! There is a similar server in PD as well. Could you figure out some way so that code can be reused? https://github.com/pingcap/pd/blob/master/pkg/dashboard/uiserver/uiserver.go